### PR TITLE
Databases Week2 Homework

### PR DESCRIPTION
**Pull request by ihk2018**
This is databases week 2 homework. Sorry for the delay.

### CODE REVIEW

File: Week2/app.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    3. Find the number of cities in which language Z is spoken (Accept Z from user)
+    4.Accept the region and language from the user.
+    5. List all the continents with the number of languages spoken in each continent
+    `);
+
+    const choices = await input(['choice']);
+    userMakeChoices = choices.choice;
+
+    const chooseFirstQuery = `SELECT name from city where id in (select capital from country where name = ?)`;
+    const chooseSecondQuery = `SELECT countrylanguage.language ,country.region FROM country JOIN countrylanguage ON countrylanguage.CountryCode = country.code WHERE country.region = ?;`;
```
Comment (mogaleaf) [inline Week2/app.js]: If you don't want to have duplicates you can also add a distinct here

File: Week2/trigger.sql [hunk 2]
```diff
@@ -0,0 +1,16 @@
+delimiter $$
+CREATE TRIGGER language_Max_Trigger BEFORE INSERT ON countrylanguage
```
Comment (mogaleaf) [inline Week2/trigger.sql]: good

Comment (ihk2018) [reply Week2/trigger.sql]: thank you

File: Week2/app.js (hunk 1 above)
Comment (ihk2018) [reply Week2/app.js]: Thank you for the valuable feedback, i will use distinct.

Comment (ihk2018) [reply Week2/app.js]: dear @mogaleaf I added the last part which is about ERD.

**Comment by Tjebbee** [comment]
@mogaleaf ; Is this one ready to be approved?

**Pull request closed without merging**